src/ansys/dynamicreporting/core/utils/report_objects.py

```diff
@@ -3473,49 +3473,129 @@ class statisticalREST(GeneratorREST):
     """
     Representation of a statistical analysis generator.
 
-    Defines getters and setters for analysis type, predictor variables, response
-    variables, and intercept.
+    Defines getters and setters for analysis type, table name, tree name, predictor
+    variables, response variables, and analysis_parameters. For variables the standard
+    input format is 2D array to support multiple variables.
     """
 
     def __init__(self):
         super().__init__()
 
-    @property
-    def analysis_type(self) -> str:
-        return self.get_property().get("analysis_type")
+    def get_analysis_type(self):
+        if "stats_params" in json.loads(self.params):
+            if "analysis_type" in json.loads(self.params)["stats_params"]:
```

Comment:
Little improvement: remove the nesting and join the conditions together with an `and`

Reply:
Thank you so much for the suggestions Visesh 😃 Updated code with the suggested improvements!